extract-initramfs-from-vmlinuz.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Does as it says on the tin.
+#
+# Example: extract-initramfs-from-vmlinuz /boot/flatcar/vmlinuz-a out-dir
+#
+# This will create out-dir/rootfs-0 directory that contains initramfs.
+
+set -euo pipefail
+
```

Comment:
```suggestion
# check for unzstd. Will abort the script with an error message if the tool is not present.
unzstd -V >/dev/null
```
I did not have the tool installed so the script failed, but with a misleading error message. Suggested improvement above.